test/opset_pass/convolution_opset_pass.cpp

```diff
@@ -180,7 +183,7 @@ TEST(opset_transform, opset1_group_convolution_backprop_data_upgrade_pass)
     auto padding_end = CoordinateDiff{3};
 
     auto group_conv_backprop = make_shared<op::v0::GroupConvolutionBackpropData>(
-        data_batch_shape, filters, delta, strides, dilations, padding_begin, padding_end, 1);
+        data_batch_shape, filters, delta, strides, dilations, padding_begin, padding_end, 4);
```

Comment:
[NON-BLOCK]
Just wonder if using `groups` variable instead of `4` value will improve readability.